apps/src/templates/sectionProgress/progressTables/ProgressTableDetailCell.jsx

```diff
@@ -84,7 +82,6 @@ export default class ProgressTableDetailCell extends React.Component {
             >
               <ProgressTableLevelBubble
                 levelStatus={subStatus}
-                isDisabled={!!level.bonus && !this.props.stageExtrasEnabled}
                 bubbleSize={progressStyles.BubbleSize.letter}
```

Comment:
now that `ProgressTableLevelBubble` doesn't need to support a disabled state, we no longer need the `stageExtrasEnabled` prop.